<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FePEST Server Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_slaves">
     <attribute name="title">
      <string>JCO-MAT-DAT</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="2">
         <widget class="QToolButton" name="toolButtonSelectMATFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditMATFilePath">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelInputFilePath_3">
          <property name="text">
           <string>MAT File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonRunJCO2MAT">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>JCO2MAT</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonRunMAT2JCO">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>MAT2JCO</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonRunMAT2DAT">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>MAT2DAT</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonRunDAT2MAT">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>DAT2MAT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QToolButton" name="toolButtonSelectJCOFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditJCOFilePath">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelInputFilePath">
          <property name="text">
           <string>JCO File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="2">
         <widget class="QToolButton" name="toolButtonSelectDATFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditDATFilePath">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelInputFilePath_4">
          <property name="text">
           <string>DAT File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>pushButtonRunMAT2JCO</zorder>
   <zorder>pushButtonRunJCO2MAT</zorder>
   <zorder>pushButtonRunMAT2DAT</zorder>
   <zorder>pushButtonRunDAT2MAT</zorder>
   <zorder>lineEditJCOFilePath</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
